content/java/concepts/math-methods/terms/sinh/sinh.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.sinh()'
+Description: 'Returns the hyperbolic sine of a double-type value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.sinh()`** method returns the hyperbolic sine of a double-type value.
+
+## Syntax
+
+```pseudo
+Math.sinh(double num)
+```
+
+The return type of the `.sinh()` method is a `double`.
+
+Some special cases for the `.sinh()` method include the following:
+
+- If the argument is `NaN`, then the result is `NaN`.
+- If the argument is infinite, then the result is an infinity with the same sign as the argument.
+- If the argument is zero, then the result is a zero with the same sign as the argument.
+
+The computed result will be within 2.5 ulps of the exact result.
+
+An ulp stands for unit of least precision. It calculates the distance between the given `double` or `float` value and the
+`double` or `float` value next larger in magnitude.
```

Comment:
```suggestion
The computed result will be within 2.5 units of least precision (ulps) of the exact result.
```

The latter part of the text is more suitable for another entry (perhaps one about the `Math.ulp()` Java method).

Reply:
Added the latter part as the description was asked by @HishamT.